From: Yaroslav Halchenko <debian@onerussian.com>
Subject: explicitly list data files to be installed if there is also subdirs

older setuptools puke:

	nipype_0.13.1-1~nd16.04+1_amd64.build:error: can't copy 'nipype/testing/data/brukerdir/pdata': doesn't exist or not a regular file
	nipype_0.13.1-1~nd80+1_amd64.build:error: can't copy 'nipype/testing/data/brukerdir/pdata': doesn't exist or not a regular file

Origin:  NeuroDebian
Last-Update: 2017-05-24

--- a/setup.py
+++ b/setup.py
@@ -100,8 +100,12 @@ def main():
         pjoin('testing', 'data', 'dicomdir', '*'),
         pjoin('testing', 'data', 'bedpostxout', '*'),
         pjoin('testing', 'data', 'tbss_dir', '*'),
-        pjoin('testing', 'data', 'brukerdir', '*'),
-        pjoin('testing', 'data', 'brukerdir', 'pdata', '*'),
+        # unfortunately older setuptools (e.g. on debian jessie and 16.04) pukes if
+        # * points also to a subdirectory.  Thus we are doomed to list all files
+        # explicitly for now if there is a also a subdirectory
+        #pjoin('testing', 'data', 'brukerdir', '*'),
+        pjoin('testing', 'data', 'brukerdir', 'fid'),
+        #pjoin('testing', 'data', 'brukerdir', 'pdata', '*'),
         pjoin('testing', 'data', 'brukerdir', 'pdata', '1', '*'),
         pjoin('workflows', 'data', '*'),
         pjoin('pipeline', 'engine', 'report_template.html'),
